Assets/MyGame/Scripts/GameClearControlScript.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+public class GameClearControlScript : MonoBehaviour
+{
+    public string ReturnSceneName;
```

Comment:
クラス外や Unity 上から操作しないので、`private` で
```suggestion
    private string ReturnSceneName;
```